build_linux_nvidia.sh

```diff
@@ -0,0 +1,381 @@
+#!/bin/bash
+# build_linux_nvidia.sh
+#   Date: 08-03-2022, 24-04-2022
+#
+#   Run this script to build MMDeploy SDK and necessary prerequisites.
+#   This script will also setup python venv
+#
+
+#####
+# Build vars
+BUILD_TYPE="Release"
+ARCH=$(uname -i)
+PROC_NUM=$(nproc)
+# Default GCC
+GCC_COMPILER="g++"
+
+#####
+# Directories
+# WORKING_DIR must correspond to script dir, i.e. MMDeploy root
+WORKING_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+PPLCV_DIR=${WORKING_DIR}/ppl.cv
+MMDEPLOY_DIR=${WORKING_DIR}/MMDeploy
+
+#####
+# Versions
+PPLCV_VER="0.6.2"
+MMDEPLOY_VER="0.4.0"
```

Comment:
Can we read the version from `mmdeploy.version.py`?

Reply:
@grimoire The MMDEPLOY_VER is only used for checking out a specific version of MMDeploy. I guess this functionality is not needed anymore and we should just use whatever version that was checked out initially.